contracts/test/Summa.ts

```diff
@@ -47,8 +33,9 @@ describe("Summa Contract", () => {
     const summa = await ethers.deployContract("Summa", [
       verifyingKey.address,
       snarkVerifier.address,
-      2, // The number of cryptocurrencies in the balance polynomials
-      14, // The number of bytes used to represent the balance of a cryptocurrency in the polynomials
+      ["ETH", "BTC"],
+      ["ETH", "BTC"],
+      8, // The number of bytes used to represent the balance of a cryptocurrency in the polynomials
```

Comment:
I believe this number refers to the number of advice columns in the range check chip in V1. Please correct me if it is wrong.
However, its relevance appears vague to me in V2.
Could you provide a bit more explanation about this?

Reply:
Do you mean that we're not using this value for any checks inside the contract?

Reply:
> Do you mean that we're not using this value for any checks inside the contract?

No, I understood this since your previous comment https://github.com/summa-dev/summa-solvency/pull/264#discussion_r1489167346

I had understood this number as representing the number of cells used in the range check circuit. which is now 4 in V2.
 However, I was a bit confused, and you're correct. It is indeed 64 bits (8 bytes). I made a mistake by calculating it in my mind. Sorry for bringing this up.

I would just like to mention here, in the comment, that the term 'balanceByteRange' signifies the maximum balance that is safe from overflow.

Reply:
Added a constructor NatSpec, also added the check of the permutations number according to balance byte range.

Reply:
Thanks for updating!